CHANGELOG.md

```diff
@@ -14,19 +14,38 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - **CUMULUS-2890**
   - Removed unused CloudWatch AWS SDK client. This change removes the CloudWatch client
     from the `@cumulus/aws-client` package.
+- **CUMULUS-3323**
+  - Updated `@cumulus/db` to by default set the `ssl` option for knex, and
```

Comment:
Do we need doc updates for this?

Reply:
Hmm, yeah that's a good point.   How about : 2ebe3b1cff3

Reply:
Great, thanks! I think this covers it and doesn't get bogged down discussing why you'd use a self-signed cert, etc.